apps/prairielearn/src/pages/instructorAssessmentOverrides/instructorAssessmentOverrides.sql

```diff
@@ -0,0 +1,63 @@
+-- BLOCK selectQuery 
+SELECT
+  COALESCE(format_date_full_compact(aap.created_at, 'America/Chicago'), '—') AS created_at,
+  COALESCE(aap.created_by::text, '—') AS created_by,
+  COALESCE(aap.credit::text, '—') AS credit,
+  COALESCE(format_date_full_compact(aap.end_date, 'America/Chicago'), '—') AS end_date,
+  COALESCE(aap.group_id::text, '—') AS group_id,
+  COALESCE(aap.note, '—') AS note,
+  COALESCE(format_date_full_compact(aap.start_date, 'America/Chicago'), '—') AS start_date,
+  COALESCE(aap.extension_type::text, '—') AS type,
+  COALESCE(aap.student_uid::text, '—') AS student_uid
+FROM
+  assessment_access_policies AS aap
+WHERE assessment_id = $assessment_id
+ORDER BY
+  aap.created_at;
+
+-- BLOCK insertQuery
+INSERT INTO assessment_access_policies
+  (assessment_id, created_at, created_by, credit, end_date, group_id, note, start_date, extension_type, student_uid)
+VALUES
+  ($assessment_id, $created_at, $created_by, $credit, $end_date, $group_id, $note, $start_date, $type, $student_uid);
+
+
+-- BLOCK updateQuery
+UPDATE assessment_access_policies
+SET
+  created_at = $created_at,
+  created_by = $created_by,
+  credit = $credit,
+  end_date = $end_date,
+  group_id = $group_id,
+  note = $note,
+  start_date = $start_date,
+  extension_type = $type,
+  student_uid = $student_uid
+WHERE
+  assessment_id = $assessment_id
+  AND (student_uid = $student_uid OR group_id = $group_id);
+
+
+-- BLOCK deleteQuery
```

Comment:
Queries conventionally use `snake_case` and are named to reflect what they do, so in this case I'd name this query something like `delete_assessment_access_policy` (ditto for elsewhere in the file).